docs/source/VUDRM-token.md

```diff
@@ -30,8 +30,8 @@ The DRM policy is included in the VUDRM token as the encrypted third component.
 | Key                | Type     | Format                    | Applicable DRM | Description                                                                                                                     |
 |--------------------|----------|---------------------------|----------------|---------------------------------------------------------------------------------------------------------------------------------|
 | `contentid`        | string   | Any                       | ALL            | The content identifier. This is used for tracking content in the stats and is not required.                                     |
-| `polbegin`         | datetime | DD-MM-YYYY HH:mm:ss       | ALL            | Policy Begin. The start date for the license. Default value depends on the DRM provider.                                                      |
-| `polend`           | datetime | DD-MM-YYYY HH:mm:ss       | ALL            | Policy End. The end date for the license. Default value depends on the DRM provider.                                                        |
+| `polbegin`         | datetime | DD-MM-YYYY HH:mm:ss       | ALL            | Policy Begin. The start date for the license. Default value depends on the DRM provider. The default timezone for this value is Europe/London. This value comes from [this database](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones) and if desired we can update your timezone to be one of the majority of these timezones.                                                    |
```

Comment:
"one of the majority"?

Reply:
I didn't think we could we could use all of the timezones in that database, only a subset?